__tests__/screens/GroupSetting/GroupSetting.test.js

```diff
@@ -1,184 +1,194 @@
+              id: userIdProfile,
+              image,
+              last_active: '2022-06-10T13:11:58.020555Z',
+              name: name,
+              online: false,
+              role: 'user',
+              updated_at: '2023-01-24T01:41:19.021868Z'
+            },
+            disabled: false,
+            frozen: false,
+            hidden: false,
+            id: channle_id,
+            last_message_at: '2023-01-24T01:00:59.432027Z',
+            member_count: 4,
+            name: 'Test group baru',
+            type: 'messaging',
+            updated_at: '2023-01-24T01:41:46.237211Z',
+            image: 'https://image.jpg'
+          },
+          disconnected: false,
+          id: channle_id,
+          initialized: true,
+          isTyping: false,
+          lastKeyStroke: undefined,
+          lastTypingEvent: null
+        },
+        profileChannel: null
+      }
+    ],
+    profile: [
+      {
+        myProfile: {
+          bio: 'Fe mobile engineer',
+          country_code: 'ID',
+          createdAt: '2022-06-10T13:11:53.000Z',
+          follower: 13,
+          follower_symbol: '< 25',
+          following: 10,
+          following_symbol: '< 25',
+          human_id: 'I4K3M10FGR78EWQQDNQ2',
+          last_active_at: '2022-06-10T13:11:53.000Z',
+          location: [
+            {
+              city: 'Yauco, PR',
+              country: 'US',
+              createdAt: location_created_at,
+              location_id: '45',
+              location_level: 'City',
+              neighborhood: '',
+              slug_name: '',
+              state: 'Puerto Rico',
+              status: 'Y',
+              updatedAt: location_created_at
             }
+          ],
+          profile_pic_asset_id: '6f47f70bea98469f4a24b6ffc550c983',
+          profile_pic_path: image,
+          profile_pic_public_id: 'pbdv3jlyd4mhmtis6kqx',
+          real_name: null,
+          status: 'Y',
+          updatedAt: '2022-07-29T12:54:04.000Z',
+          user_id: userIdProfile,
+          username: name
         }
+      }
+    ]
+  };
+  const wrapper = ({children}) => (
+    <Context.Provider value={mockContext}>{children}</Context.Provider>
+  );
 
+  it('Should match snapshot', () => {
+    const navigation = {
+      push: jest.fn()
+    };
+    const route = {
+      params: {
+        focusChatName: 'Settings'
+      }
+    };
 
-        const {toJSON} = render(<GroupSetting navigation={navigation} route={route} />, {wrapper})
-        expect(toJSON).toMatchSnapshot()
-    })
-
-    it('onPress heaeder should run correctly', () => {
-         const navigation = {
-            push: jest.fn(),
-            goBack: jest.fn()
-        }
-        const route = {
-            params: {
-                focusChatName: 'Settings'
-            }
-        }
-
+    const {toJSON} = render(<GroupSetting navigation={navigation} route={route} />, {wrapper});
+    expect(toJSON).toMatchSnapshot();
```

Comment:
should be 
`expect(toJSON()).toMatchSnapshot();`

toJSON should be invoked --> toJSON() so it return the correct snapshot on .snap file